### ISSUE TITLE
Url bar is not hidden in the android version upper to 10

### ISSUE THREAD

**Issue by SaeedNoshadi89** [GoogleChrome/android-browser-helper#360]
Url bar is not hidden in the android version upper to 10. I checked everything and I didn't find any issue.
**I checked SHA-256 in the google console and assetlinks.json**.

**Comment by PEConn** [comment]
So you're finding that the URL bar is hidden on Android N/O/P, but that it appears on Android 10/11/12?

**Comment by SaeedNoshadi89** [comment]
Yes. but I can't see the URL bar in android 10 and I see it in the android version bigger than 10.

Everything is fine in android 10 and less.

**Comment by PEConn** [comment]
Could you provide your APK?

**Comment by SaeedNoshadi89** [comment]
@PEConn  Sure

https://play.google.com/store/apps/details?id=com.mykuyauserapp

**Comment by danielm2402** [comment]
@SaeedNoshadi89 Hello, how could you solve this?
